TabNavigator.js

```diff
@@ -95,21 +95,31 @@ export default class TabNavigator extends React.Component {
   }
 
   _renderTab(item) {
-    let icon;
     if (item === null) {
       return;
     }
-    if (item.props.selected) {
+
+    let icon;
+    let selectedIcon;
+    if (item.props.renderIcon) {
+      // Always render both icons, using opacity to control which one is visible
+      // this prevents flickering when using images in Android
+
+      icon = item.props.renderIcon();
       if (item.props.renderSelectedIcon) {
```

Comment:
this looks a big funky that its nested inside if there's a renderIcon or not, might be better to put it below instead